changelog.txt

```diff
@@ -0,0 +1,2 @@
+- important - [breaking change] The Firebase Functions SDK no longer supports Node 6. Developers must use Node 8.13.0 or higher.
+- important - [breaking change] The Firebase Functions SDK no longer supports Firebase Tools SDK below major version 5.
```

Comment:
The CLI implemented support to deploy Node 10 functions in 6.8.0. Do you think we should use that version instead? That way, they have two runtimes to choose from, instead of just having 8. Technically it's not breaking though, so should work with v5 (btw, why did you choose v5? I saw `FIREBASE_PROJECT` was used in v3 so wondering what else was there?)

Reply:
My intention was to choose the latest major version, I must have looked at the wrong `package.json`. I most definitely don't mind bumping it, but should we go for `^6.0.0` or `^6.8.0`? The fact that it's _supported_ does not imply that it is _required_. WDYT?

Reply:
Let's do `^6.8.0` since that has the functionality to deploy to Node 10, and generally brings us up to par with current node versions. I think it's better to offer this with ability to deploy to 10 than having someone realize later that they cannot deploy to 10 with the version they're on.

Reply:
Let's do ^6.8.0 since that has the functionality to deploy to Node 10, and generally brings us up to par with current node versions. I think it's better to offer this with ability to deploy to 10 than having someone realize later that they cannot deploy to 10 with the version they're on.

Reply:
bump on this

Reply:
Done in 31b2d9e0ec65533fab0e156fa3d56d174d31aa01